build.sbt

```diff
@@ -19,8 +19,8 @@ libraryDependencies ++= {
   val json4sVersion = "3.5.0"
 
   Seq(
-    "org.apache.kafka" % "connect-api" % kafkaVersion,
-    "org.apache.kafka" % "connect-json" % kafkaVersion,
+    "org.apache.kafka" % "connect-api" % kafkaVersion % "provided",
+    "org.apache.kafka" % "connect-json" % kafkaVersion % "provided",
     "com.microsoft.azure" % "azure-eventhubs" % azureEventHubSDKVersion,
```

Comment:
does the provided package match the version required? I'm wondering what happens if we expect version "foo.X" and the provided is "foo.Y"